### ISSUE TITLE
Google Integration No longer exists

### ISSUE THREAD

**Issue by mrzoops** [d8ahazard/FlexTV#27]
### IMPORANT, PLEASE READ FIRST:

Please read over the [Notes/FAQ](Notes-FAQ) and [networking](Talking-To-The-Outside-World) sections before reporting an issue.

If your issue is not resolved, try to fill out the following as completely as possible. I'm not going to delete issues just because forms aren't filled out completely, but I will if you provide me no information whatsoever.

If submitting a feature request, these are not required.

#### 6. If you're having issues with a specific command, please note the timestamp in the web UI, and take a look at Flex TV and Flex TV_error.log files.  Paste anything that looks related as well.

*BE SURE TO CHANGE ANY LINES CONTAINING API TOKENS OR OTHER PERSONAL IDENTIFYING INFORMATION*

#### 8. Please describe the issue.
FlexTV no longer avaialble from google. No way to create the connection. Following this link results in the service being removed possibly?

https://assistant.google.com/services/a/uid/00000000e3faabc3?hl=en-US

**Comment by d8ahazard** [comment]
Yeah...there's some issue with Dialogflow at the moment, I've opened a
critical support case with them to get it fixed...

**Comment by kverkind** [comment]
@d8ahazard can we get an update please?

**Comment by d8ahazard** [comment]
> @d8ahazard can we get an update please?

I'm in a special hell with Google/DialogFlow support right now...

They tell me I need to re-integrate my agent with Google to fix it. Only when I try to log into DialogFlow to do ANYTHING, I get a loop that makes me continuously re-auth to Google. 

So, I can't sign in, and Google is being slow AF in helping.

Honestly, I'm getting more and more frustrated with this project as time goes by...and the fact that it's competing with other paying projects doesn't help...

**Comment by sintaxx** [comment]
competing paying projects? any names? i'd sooner pay you but i'm curious what's out there

**Comment by mrzoops** [comment]
Yeah I was kind of interested in the same thing. This is the best project I've seen and I would pay for it, but I would be interested in additional projects if there are any.

**Comment by evanrich** [comment]
so wait, i just found out about this project...is it DOA now?

**Comment by sintaxx** [comment]
@evanrich I don't believe so. He's just having issues with Google integrations afaik

**Comment by d8ahazard** [comment]
it's not DOA. I'm having some kind of extreme problems with dialog flow
right now. Google has been investigating it for the past couple days, but
there's nothing I can do to make it work until then.

**Comment by rieger07** [comment]
@d8ahazard Hope you're hangin in there bro. Any news? <3

**Comment by Shinkiro1** [comment]
@d8ahazard Ping.  Hope all is well.  Those of us monitoring the thread on https://forums.plex.tv/t/flex-tv-back-online-google-assistant-integration-for-plex/174627/2365 would really appreciate an update, even if the update is that there hasn't been any progress yet.

If there's anything we can do to help, financially or otherwise, please let us know.

**Comment by colonelmattyman** [comment]
@d8ahazard has links to his Patreon and PayPal at the bottom of the main page on GitHub.

https://github.com/d8ahazard/FlexTV

**Comment by mrzoops** [comment]
Gladly will leave a contribution if everything gets working again.

**Comment by intertan** [comment]
@d8ahazard can you keep us updated with the status with google? I hope this project doesn't need to go DOA due to google

**Comment by madboykiller** [comment]
any progress on the google Integration its a real bum if thats not going to work anymore

**Comment by LoudSoftware** [comment]
Any way to host our own dialogflow project?
I would love to use this project again but it seems like google is taking their sweet time.

**Comment by tsheridan-r7** [comment]
Monthly check-in :)

Any update on this after the recent Google Home changes? This was a great exercise in learning a docker deployment, finally got to the point to send a voice command when I saw this issue thread.

Hope functionality is restored sometime soon, great work on this!

**Comment by LoudSoftware** [comment]
Google home integration is presently broken, as DialogFlow changed their API version to V2, and I've yet to fully update the application yet. I'll get it fixed when I have time.

_Originally posted by @d8ahazard in https://github.com/d8ahazard/FlexTV/issues/28#issuecomment-549145064_

**Comment by crnarendran** [comment]
I would love to use this with Google Home, but looks like integration is still broken. Will this be fixed?

**Comment by scloutier75** [comment]
+1

**Comment by d8ahazard** [comment]
Soon. :D

**Comment by beirtipol** [comment]
Stay strong. I used this a while ago but only recently got a Google nest speaker and pulled a sad face when I saw it wasn't working. Anything you need help with?

**Comment by d8ahazard** [comment]
Honestly...and don't scream...it should be fixed. I'm just hammered with
stuff to do, and simply need to make sure my testing account for Flex TV is
set up right, and then re-submit to google for approval.

I should do that today...

**Comment by scloutier75** [comment]
Awesome!  Is there any way it could work all installed locally or we will always be vulnerable to a change in the APIs that could break things apart?

**Comment by d8ahazard** [comment]
it uses the same APIs no matter where it's hosted. However I don't think
they're going to be upgrading again for a while.

**Comment by savaloi** [comment]
Epic man, thanks soo much for your hard work! I look forward to installing this

**Comment by scloutier75** [comment]
Any news?  Don't want to put any pressure on you but I am really looking forward to try this...

**Comment by kverkind** [comment]
Pressure is not the good word here.

**Comment by Fame89** [comment]
@d8ahazard Seems correct assuming the Google Home integration is still not working?

**Comment by beyondmeat** [comment]
We're all stuck at home, we need this!

**Comment by Orhelien** [comment]
Any news ?

**Comment by JacobOkanta** [comment]
It's been quite some time since this issue showed up, is this a dead project or is there some update?

**Comment by d8ahazard** [comment]
Not dead...just don't have the time to work on it, presently.

I got most of the V2 Dialogflow API stuff done, but ran into some server
issues when it came to the authorization flow, and just haven't been able
to work on it.

If someone out there is knowledgeable with PHP and trustworthy, I'd be
willing to discuss granting access to the server code to get this bit fixed
up.
